torchtune/utils/metric_logging.py

```diff
@@ -138,6 +138,7 @@ class WandBLogger(MetricLoggerInterface):
         project (str): WandB project name
         entity (Optional[str]): WandB entity name
         group (Optional[str]): WandB group name
+        log_dir (Optional[str]): WandB log directory
```

Comment:
nit: should explain what's used if this is not specified (or if dir is specified)